code/game/objects/items/weapons/power_cells.dm

```diff
@@ -103,16 +122,15 @@
 	minor_fault = 1
 	rating = 1
 
-/obj/item/weapon/stock_parts/cell/slime
+/obj/item/weapon/stock_parts/cell/high/slime
 	name = "charged slime core"
 	desc = "A yellow slime core infused with plasma, it crackles with power."
 	origin_tech = "powerstorage=2;biotech=4"
-	icon = 'icons/mob/slimes.dmi' //'icons/obj/harvest.dmi'
-	icon_state = "yellow slime extract" //"potato_battery"
-	maxcharge = 10000

```

Comment:
Why exactly are xenobio's slime cells being reduced by a factor of ten? Is this a balance issue somehow? I fail to see how limited production of High-capacity powercells is so bad.